tests/system/small/test_dataframe_io.py

```diff
@@ -317,6 +317,51 @@ def test_to_gbq_w_None_column_names(
     )
 
 
+@pytest.mark.parametrize(
+    "clustering_columns",
+    [
+        pytest.param(["int64_col", "geography_col"]),
+        pytest.param(
+            ["float64_col"],
+            marks=pytest.mark.xfail(raises=google.api_core.exceptions.BadRequest),
```

Comment:
Worth adding one more testcase with duplicate clustering column names throwing the exception

Reply:
Done.